<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuestGUIWidget</class>
 <widget class="QWidget" name="GuestGUIWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GuestGUIWidget</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>502</width>
     <height>436</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="userInformationTableView">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>80</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="bookInformationTableView">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>200</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>200</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="searchBookLineEdit">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>30</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>book name</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="searchingButton">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Searching</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="borrowButton">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>borrow</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="returnButton">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>return</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="logOffButton">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>log off</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="exitButton">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
